src/pages/evaluations/template.py

```diff
@@ -0,0 +1,181 @@
+import json
+
+import streamlit as st
+
+# from inspect_evals_scoring.process_log import DashboardLog
+from src.log_utils.dashboard_log import DashboardLog
+from src.log_utils.dashboard_log_utils import get_metrics
+from src.plots.bar import create_bar_chart
+from src.plots.cutoff_scatter import create_cutoff_scatter
+from src.plots.pairwise import create_pairwise_analysis_table
+from src.plots.plot_utils import highlight_confidence_intervals
+
+
+def render_page(eval_logs: list[DashboardLog]):
+    st.subheader("Naive cross-model comparison")
+    st.markdown("""
+                Graphs in this section compare how different AI models perform on the same task. We call this a "naive" comparison because it uses simple averages to compare models, which doesn't tell us if one model is statistically significantly better than another. For more reliable comparisons, check out the next section. To get more accurate scores, we evaluate each sample in a dataset multiple times using the epochs feature in Inspect AI.
+                """
+    )
+
+    col1, col2, col3, col4 = st.columns(4)
+
+    with col1:
+        naive_task_name = st.selectbox(
+            "Evaluation/task",
+            sorted(set(log.eval.task for log in eval_logs)),
+            index=0,
+            format_func=lambda option: option.removeprefix("inspect_evals/"),
+            help="Name of the evaluation and the task",
+            label_visibility="visible",
+            key="cross_model_comparison_task_name",
+        )
+
+    naive_logs = [log for log in eval_logs if log.eval.task == naive_task_name]
+
+    with col2:
+        model_provider = st.selectbox(
```

Comment:
I think it's likely the user want to be able to select multiple of each model/provider/etc. instead of only one (i.e. multi-select, like when in an e-commerce interface)

Reply:
[st.multiselect](https://docs.streamlit.io/develop/api-reference/widgets/st.multiselect) is a good alternative to address this cc. @eleweek